lib/npm/test.js

```diff
@@ -55,8 +55,14 @@ function test(context, next) {
       npmBin = path.join(path.dirname(npmBin), 'node_modules', 'npm', 'bin',
           'npm-cli.js');
     }
-    const proc = spawn(nodeBin, [npmBin, 'test', '--loglevel=' +
-        context.options.npmLevel], options);
+
+    let args = [npmBin, 'test', '--loglevel=' + context.options.npmLevel];
+
+    if (context.options.customTest)  {
+        args = context.options.customTest.split(' ');
```

Comment:
As I mentioned before, I don't think we should allow people to pass in multiple args, we don't want to get into the business of splitting args in a way that shells can parse. If we only allow a single argument that is the path to the file, then we can handle spaces in paths by default.

tl;dr remove the `split(' ')`.